Source/CsDebugScript.Engine/Engine/Native/DEBUG_LAST_EVENT_INFO.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace CsDebugScript.Engine.Native
+{
+    public enum DEBUG_EVENT
+    {
+        /// <summary>
+        /// A breakpoint exception occurred in the target.
+        /// </summary>
+        DEBUG_EVENT_BREAKPOINT,
```

Comment:
Btw, is this enum defined in DbgEng.h? If so, you can change CreateDbgEngIdl program to extract it for you. Take a look at:
                WriteConstants(outputString, usedConstants, ref remainingConstants, "DEBUG_EXECUTE_");